test/JoeMakerV4.test.ts

```diff
@@ -293,7 +296,7 @@ describe("joeMakerV4", function () {
       expect(await this.joe.balanceOf(BAR_ADDRESS)).to.equal("90009160457674355334497135")
 
       await this.avaxUsdc.transfer(this.joeMakerV4.address, await this.avaxUsdc.balanceOf(this.dev.address))
-      await this.joeMakerV4.convert(this.usdc.address, this.wavax.address)
+      await this.joeMakerV4.convert(this.usdc.address, this.wavax.address, "100")
       expect(await this.joe.balanceOf(BAR_ADDRESS)).to.equal("90009210905950105791950559")
```

Comment:
What's the difference increase in JOE? Let's assert that it equals to that instead of a random number